### PR TITLE
Fix old-console/cluster.html; Remove redundant throws in ITBasicAuthConfigurationTest

### PR DESCRIPTION
**Pull request by leventov**
Continue efforts to try to resolve problems that prevent TeamCity from updating to IntelliJ 2018.2 engine, started in #7499 and #7541.

### CODE REVIEW

File: web-console/old-console/cluster.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
             <input type="submit" value="Move" />
           </form>
         </div>
-        <!--<div id="drop">
+        <div id="drop">
```
Comment (vogievetsky) [inline web-console/old-console/cluster.html]: How come you are only removing one side of the comment? I have no idea what this is about but should you also remove the `-->` on line 67?

Comment (vogievetsky) [reply web-console/old-console/cluster.html]: Or is your fix that this comment is unbalanced?

Comment (gianm) [reply web-console/old-console/cluster.html]: It looks like it's a comment inside another comment. Might as well just delete the whole thing, I suppose.

Comment (egor-ryashin) [reply web-console/old-console/cluster.html]: @leventov could you delete the whole comment?

Comment (leventov) [reply web-console/old-console/cluster.html]: Deleted.

**Pull request merged**